docs/reference/readme.md

```diff
@@ -601,6 +601,16 @@ See also:
 
 Authenticate Service URL is the externally accessible URL for the authenticate service.
 
+### Signout Redirect URL
+
+- Environmental Variable: `SIGNOUT_REDIRECT_URL`
+- Config File Key: `signout_redirect_url`
+- Type: `URL`
+- Required
+- Example: `https://signout-redirect-url.corp.example.com`
+
+Signout redirect url is the url user will be redirected to after signing out.
```

Comment:
Can we also add a note here saying it is also possible set a redirect url by passing in a custom queryparam / post value when hitting the `./pomerium/signout` endpoint?

Reply:
ok